sims/benchmark-simuPOP.py

```diff
@@ -237,31 +234,37 @@ def __call__(self, loc, alleles):
 rc = RecombCollector(ts=init_ts, node_ids=node_ids,
                      locus_position=locus_position,
                      benchmark=True, mode='binary')
-
+pre_ops = [sim.PyOperator(lambda pop: rc.increment_time() or True)]
+mating_ops = [id_tagger]
+post_ops = []
```

Comment:
If you put some periodic output into `post_ops` we could at least tell how far it had gotten when it got killed, if it does?